### PR TITLE
Add Pilz Industrial Motion Planner tutorial (#461)

### PR DESCRIPTION
**Pull request by martiniil**
### Description

Melodic backport adding the pilz_industrial_motion_planner tutorial (#461)

## Prerequisites
- [ ] https://github.com/ros-planning/moveit/pull/2507 is merged

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

**Review by jschleicher — APPROVED** [review]
👍 
Htmlproofer should get green after the merge of ros-planning/moveit#2507

File: doc/pilz_industrial_motion_planner/pilz_industrial_motion_planner.rst [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+To use the command planner cartesian limits have to be defined. The
+limits are expected to be under the namespace
+``<robot_description>_planning``. Where ``<robot_description>`` refers
+to the parameter under which the URDF is loaded. E.g. if the URDF was
+loaded into ``/robot_description`` the cartesian limits have to be
+defined at ``/robot_description_planning``.
+
+An example showing the cartesian limits which have to be defined can be
+found in `prbt_moveit_config
+<https://github.com/ros-planning/moveit_resources/blob/melodic-devel/prbt_moveit_config/config/cartesian_limits.yaml>`_.
```
Comment (davetcoleman) [inline doc/pilz_industrial_motion_planner/pilz_industrial_motion_planner.rst:295]:
```suggestion
<https://github.com/ros-planning/moveit_resources/blob/master/prbt_moveit_config/config/cartesian_limits.yaml>`_.
```
```suggestion
<https://github.com/ros-planning/moveit_resources/blob/melodic-devel/prbt_moveit_config/config/cartesian_limits.yaml>`_.
```

Comment (davetcoleman) [reply doc/pilz_industrial_motion_planner/pilz_industrial_motion_planner.rst:295]: This is causing a broken link in Travis

**Review by davetcoleman — APPROVED** [review]
Pls accept my suggestion to retrigger travis build

File: doc/pilz_industrial_motion_planner/pilz_industrial_motion_planner.rst (hunk 1 above)
Comment (martiniil) [reply doc/pilz_industrial_motion_planner/pilz_industrial_motion_planner.rst:295]: Thank you.

**Pull request merged**